evm-tests/ethcore-builtin/src/lib.rs

```diff
@@ -531,6 +537,7 @@ pub type Bls12MultiexpPricerG2 = Bls12MultiexpPricer<G2Marker>;
 ///
 /// Call `cost` to compute cost for the given input, `execute` to execute the contract
 /// on the given input, and `is_active` to determine whether the contract is active.
+#[derive(Debug)]
```

Comment:
As I understand, the derivation of the `Debug` we need for test purposes, right? If so, does it make sense to change it to: 
 ```suggestion
#[cfg_attr(test, derive(Debug))]
```
?
The point is not to include it into wasm binary, which could decrease the contract size.

Reply:
It's related only to `evm-tests`.